src/frontend/src/app/startpage/startpage.component.scss

```diff
@@ -0,0 +1,116 @@
+.header {
+  display: flex;
+  justify-content: center;
+  overflow-y: hidden;
+  height: auto;
+}
+.iconcontainer {
+  display: flex;
+  justify-content: center;
+  margin-top: 100px;
+}
+.center {
+  display: flex;
+  justify-content: center;
+  overflow-y: hidden;
+  height: auto;
+}
+.share {
+  background-color: white;
+  color: black;
+  transform: scale(2);
+}
+
+.MuiBox-root {
+  align-items: center;
+}
+
+.roundbutton {
+  border-radius: 20px;
+  background: transparent;
+  padding: 6px 12px;
+  width: 400px;
+  max-width: 80vw;
+  height: 55px;
+  text-align: center;
+  font-size: 30px;
+  letter-spacing: 2px;
+  color: #ffffff;
+  background-color: #000000;
+  font-weight: 800;
+  margin: 16px;
+  text-transform: uppercase;
+  cursor: pointer;
+  border: 7px solid;
+  border-color: #000000;
+}
+
+.loginbutton {
+  margin: 16px;
+  border-radius: 20px;
+  background-color: #000000;
+  color: #ffffff;
+  height: 55px;
+  display: flex;
+  align-items: center;
+  font-weight: 800;
+  font-size: 30px;
+  overflow: hidden;
+  padding: 10px;
+  cursor: pointer;
+  text-transform: uppercase;
+}
+
+.logintext {
+  margin-left: 50px;
+}
+
+.loginicon {
+  margin-right: 20px;
+  width: 40px;
+  flex-shrink: 0;
+  height: 40px;
+  background-color: #ffffff;
+  border-radius: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #ffffff;
+  text-transform: uppercase;
+}
+
+.personbackground {
+  width: 0.9em;
+  height: 0.9em;
+  display: inline-block;
+  fill: white;
+  background-color: #000000;
+  font-size: 2.1875rem;
+}
+
+.rightarrow {
+  color: white;
+  background-color: white;
+}
+
+.buttoncontainer {
+  position: relative;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
```

Comment:
Don't have to worry about the webkit, ms stuff for flex box. Flexbox widely supported on all browsers now

Reply:
Removed webkit and ms flexbox